### PR TITLE
Added context string consideration

### PR DESCRIPTION
**Pull request by feventura**
Added context string consideration, as defined in #35.

### CODE REVIEW

**Comment by ounsworth** [comment]
Do not merge, we decided to do this differently. See comments #42

**Comment by janklaussner** [comment]
Closes #42

File: draft-ietf-lamps-pq-composite-sigs.md [hunk 1]
```diff
@@ -316,43 +316,192 @@ The key generation functions MUST be executed for both algorithms. Compliant par
...
                         composite inputs.
 
      A1, A2             Component signature algorithms. See note below on
                         composite inputs.
 
-     Message            The Message to be signed, an octet string
+     M                  The Message to be signed, an octet string
+
+     PH                 The Message Digest Algorithm for pre-hashing.  See
+                        section on pre-hashing the message below.
```
Comment (ounsworth) [inline draft-ietf-lamps-pq-composite-sigs.md:492]: Which "section on pre-hashing the message" is this referring to? I don't see one.

**Review by ounsworth — CHANGES_REQUESTED** [review]

**Comment by johngray-dev** [comment]
All the changes in this pull request are now in #59 which will eventually get pushed to main

**Pull request closed without merging**